manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -212,7 +212,10 @@ spec:
         securityContext:
           privileged: false
           runAsGroup: 1337
-          runAsUser: 0
+          runAsUser: 1337
```

Comment:
Is running as a specific UID/GUID even important for non-sidecar envoys? I wouldn't expect so.

Reply:
No we just need a non-root one really. We already do this for the other gateways so if we should change them lets do it all together?